src/client/TabList/index.tsx

```diff
@@ -30,7 +30,7 @@ export const TabList = ({
           closeTabs([activeFileId]);
         }
         if (event.key == 'n') {
-          createFile();
+          createFile('UnnamedFile');
```

Comment:
Interesting.... This will actually write the file to the file system, an will set up the editor with no file extension.

Good for a first pass! Related to #31 (that will need to happen for syntax highlighting to be correct after a the file gets renamed with a new extension)